ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -1139,7 +1139,7 @@ const triggerSet: TriggerSet<Data> = {
       netRegexJa: NetRegexes.abilityFull({ source: ['アイスクロウ', 'サンダーウィング', 'ライトファング', 'ダークテイル', 'ファイアホーン'], id: ['26C6', '26C7', '26CA', '26C9', '26C5'] }),
       netRegexCn: NetRegexes.abilityFull({ source: ['冰爪', '雷翼', '光牙', '暗尾', '火角'], id: ['26C6', '26C7', '26CA', '26C9', '26C5'] }),
       netRegexKo: NetRegexes.abilityFull({ source: ['얼음발톱', '번개날개', '빛의 송곳니', '어둠의 꼬리', '화염뿔'], id: ['26C6', '26C7', '26CA', '26C9', '26C5'] }),
-      condition: (data, matches) => !data.seenDragon || !(matches.source in data.seenDragon),
+      condition: (data, matches) => !(matches.source in data.seenDragon),
```

Comment:
Hmmm this does seem like a logic change here in that previously this would have always run unconditionally (unintentionally).  I thiiiink it should still work though?

Reply:
The conditional here was a no-op, as outlined elsewhere `(!{}) === false`, it looks like it was just an `undefined` check before, not relying on false-y empty object behavior.

Reply:
Oh, you're right you're right.  I was misreading this as `true || etc` and not `false || etc`, sorry.